securedrop_client/api_jobs/uploads.py

```diff
@@ -43,21 +43,27 @@ def call_api(self, api_client: API, session: Session) -> str:
             session.add(reply_db_object)
             session.commit()
             return reply_db_object.uuid
-        except CryptoError as ce:
-            message = "Failed to encrypt reply for source {id} due to CryptoError: {error}".format(
-                id=self.source_uuid, error=ce)
-            raise SendReplyJobException(message, self.reply_uuid)
+        except RequestTimeoutError as e:
+            message = "Failed to send reply for source {id} due to Exception: {error}".format(
+                id=self.source_uuid, error=e)
+            raise SendReplyJobTimeoutError(message, self.reply_uuid)
         except Exception as e:
             message = "Failed to send reply for source {id} due to Exception: {error}".format(
                 id=self.source_uuid, error=e)
-            raise SendReplyJobException(message, self.reply_uuid)
+            raise SendReplyJobError(message, self.reply_uuid)
 
     def _make_call(self, encrypted_reply: str, api_client: API) -> sdclientapi.Reply:
         sdk_source = sdclientapi.Source(uuid=self.source_uuid)
         return api_client.reply_source(sdk_source, encrypted_reply, self.reply_uuid)
 
 
-class SendReplyJobException(Exception):
+class SendReplyJobError(Exception):
+    def __init__(self, message: str, reply_uuid: str):
+        super().__init__(message)
+        self.reply_uuid = reply_uuid
+
+
+class SendReplyJobTimeoutError(Exception):
```

Comment:
What do you think about this exception being a subclass of `RequestTimeoutError`? Then below in the queue logic we can replace `(RequestTimeoutError, SendReplyJobTimeoutError)` -> `RequestTimeoutError` to catch the base timeout exception